demos/common/CMakeLists.txt

```diff
@@ -2,19 +2,4 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-find_package(InferenceEngine 2.0 REQUIRED)
-find_package(OpenCV REQUIRED COMPONENTS core imgcodecs videoio)
-
-add_subdirectory(monitors)
-add_subdirectory(models)
-add_subdirectory(pipelines)
-
-file(GLOB_RECURSE HEADERS "${CMAKE_CURRENT_SOURCE_DIR}/include/*")
-file(GLOB_RECURSE SOURCES "${CMAKE_CURRENT_SOURCE_DIR}/src/*")
-
-source_group("src" FILES ${SOURCES})
-source_group("include" FILES ${HEADERS})
-
-add_library(common STATIC ${HEADERS} ${SOURCES})
-target_include_directories(common PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/include")
-target_link_libraries(common PRIVATE gflags ${InferenceEngine_LIBRARIES} opencv_core opencv_imgcodecs opencv_videoio)
+add_subdirectory(cpp)
```

Comment:
This file seems unnecessary. You can just directly use `add_subdirectory(common/cpp)` in `demos/CMakeLists.txt`.

Reply:
Done